examples/cfd/external_aerodynamics/domino/README.md

```diff
@@ -0,0 +1,15 @@
+# DoMINO: Decomposable Multi-scale Iterative Neural Operator for External Aerodynamics
+
+DoMINO is a local, multi-scale, point-cloud based model architecture to model large-scale 
+physics problems such as external aerodynamics. The DoMINO model architecture takes STL 
+geometries as input and evaluates flow quantities such as pressure and 
+wall shear stress on the surface of the car as well as velocity fields and pressure 
+in the volume around it. The DoMINO architecture is designed to be a fast, accurate 
+and scalable surrogate model for large-scale simulations.
+
+The DoMINO model architecture is used to support the Real Time Wind Tunnel OV Blueprint 
+demo presented at Supercomputing' 24.
+
+<img width="2372" alt="image" src="https://github.com/user-attachments/assets/c653d477-c765-43a5-9ad7-09b21472c584">
```

Comment:
I think rather than adding it this way, we can add the .png to the `img` folder or maybe to the docs folder here https://github.com/NVIDIA/modulus/tree/main/docs/img.